dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task210908AddCompanyDefaultLanguageTest.java

```diff
@@ -0,0 +1,132 @@
+package com.dotmarketing.startup.runonce;
+
+
+import static com.dotmarketing.portlets.languagesmanager.business.LanguageFactoryImpl.DEFAULT_LANGUAGE_CODE;
+import static com.dotmarketing.portlets.languagesmanager.business.LanguageFactoryImpl.DEFAULT_LANGUAGE_COUNTRY_CODE;
+import static com.dotmarketing.util.ConfigUtils.*;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.portlets.languagesmanager.business.LanguageAPI;
+import com.dotmarketing.portlets.languagesmanager.model.Language;
+import com.dotmarketing.util.Config;
+import com.dotmarketing.util.ConfigUtils;
+import com.dotmarketing.util.Logger;
+import io.vavr.Tuple2;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class Task210908AddCompanyDefaultLanguageTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
+        // Setting web app environment
+        IntegrationTestInitService.getInstance().init();
+    }
+
+    private void dropColumn(final DotConnect dotConnect) throws DotDataException{
+
+        Logger.debug(this, "Prepping for testing `add` column.");
+        try {
+            DbConnectionFactory.getConnection().setAutoCommit(true);
+            final String dropColumnSQL = "ALTER TABLE company DROP COLUMN default_language_id";
```

Comment:
I think this drop will break the first time if the column doesn't exist

Reply:
true